bundles/io/org.eclipse.smarthome.io.net/src/main/java/org/eclipse/smarthome/io/net/http/HttpUtil.java

```diff
@@ -74,7 +76,7 @@
      * @return the response body or <code>NULL</code> when the request went wrong
      * @throws IOException when the request execution failed, timed out or it was interrupted
      */
-    public static String executeUrl(String httpMethod, String url, int timeout) throws IOException {
+    public static String executeUrl(String httpMethod, String url, long timeout) throws IOException {
```

Comment:
Does this break compatibility with existing jar's on the marketplace, is it really needed? A timeout of more then 2.000.000 looks too high to me anyway.

Reply:
I made the mistake of thinking that max int value was 32767 !
I will return to int type.